libs/web/shell/ui/user-dropdown/src/lib/user-dropdown.store.ts

```diff
@@ -1,16 +1,18 @@
-import { ComponentStore } from '@ngrx/component-store';
 import { AuthStore } from '@angular-spotify/web/auth/data-access';
-import { Observable } from 'rxjs';
 import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { Observable } from 'rxjs';
 
 @Injectable()
 export class UserDropdownStore extends ComponentStore<Record<string, string>> {
```

Comment:
I don't think think we need a store for this component, can we just inject the `AuthStore` directly into the component and take the value from there?

Reply:
Hi @trungk18,

Yes we can, but I just followed your steps as you brought userAvatar and userName from AuthStore to here and then inject this store to component.

I will modify this too. Also Do you want me to bring userName and userAvatar directly from AuthStore to component ?